syntax = "proto3";

package envoy.extensions.transport_sockets.raw_buffer.v3alpha;

import "udpa/annotations/versioning.proto";

option java_package = "io.envoyproxy.envoy.extensions.transport_sockets.raw_buffer.v3alpha";
option java_outer_classname = "RawBufferProto";
option java_multiple_files = true;

// [#protodoc-title: Raw Buffer]
// [#extension: envoy.transport_sockets.raw_buffer]

// Configuration for raw buffer transport socket.
message RawBuffer {
  option (udpa.annotations.versioning).previous_message_type =
      "envoy.config.transport_socket.raw_buffer.v2.RawBuffer";
}
